dist/platforms/ubuntu/entrypoint.sh

```diff
@@ -1,47 +1,63 @@
 #!/usr/bin/env bash
 
-#
-# Create directory for license activation
-#
-
-ACTIVATE_LICENSE_PATH="$GITHUB_WORKSPACE/_activate-license~"
-mkdir -p "$ACTIVATE_LICENSE_PATH"
-
-#
-# Run steps
-#
-source /steps/set_extra_git_configs.sh
-source /steps/set_gitcredential.sh
-source /steps/activate.sh
-source /steps/build.sh
-source /steps/return_license.sh
-
-#
-# Remove license activation directory
-#
-
-rm -r "$ACTIVATE_LICENSE_PATH"
-chmod -R 777 "/BlankProject"
-
-#
-# Instructions for debugging
-#
-
-if [[ $BUILD_EXIT_CODE -gt 0 ]]; then
-echo ""
-echo "###########################"
-echo "#         Failure         #"
-echo "###########################"
-echo ""
-echo "Please note that the exit code is not very descriptive."
-echo "Most likely it will not help you solve the issue."
-echo ""
-echo "To find the reason for failure: please search for errors in the log above."
-echo ""
-fi;
-
-#
-# Exit with code from the build step.
-#
-
-exit $BUILD_EXIT_CODE
+fullProjectPath="$GITHUB_WORKSPACE/$PROJECT_PATH"
+
+# Get host user/group info so we create files with the correct ownership
+USERNAME=$(stat -c '%U' "$fullProjectPath")
+USERID=$(stat -c '%u' "$fullProjectPath")
+GROUPNAME=$(stat -c '%G' "$fullProjectPath")
+GROUPID=$(stat -c '%g' "$fullProjectPath")
+
+groupadd -g $GROUPID $GROUPNAME
+useradd -u $USERID -g $GROUPID $USERNAME
+usermod -aG $GROUPNAME $USERNAME
+mkdir -p "/home/$USERNAME"
+chown $USERNAME:$GROUPNAME "/home/$USERNAME"
```

Comment:
Since the script doesn't start with `set -x`, shoud we add some error handling for these 5 lines here?

Reply:
Good point, will add some error checking. Also realized I can put this inside the later if statement so it only runs when using the run as host option

Reply:
I wrote `set -x`, but I meant `set -e`, I edited my previous comment. I see this is resolved in [91ab33e](https://github.com/game-ci/unity-builder/pull/600/commits/91ab33e43b95154901ff0a03dfeb57db3f948ea5)